guides/source/association_basics.md

```diff
@@ -1258,7 +1258,7 @@ Controls what happens to the associated object when its owner is destroyed:
 
 * `:destroy` causes the associated object to also be destroyed
 * `:delete` causes the associated object to be deleted directly from the database (so callbacks will not execute)
-* `:nullify` causes the foreign key to be set to `NULL`. Callbacks are not executed.
+* `:nullify` causes the foreign key to be set to `NULL`. Callbacks are not executed. On polymorphic associations both the foreign key and `*_type` column will be nullified.
```

Comment:
Please also update the associated commented documentation in `activerecord/lib/active_record/associations.rb`.

Reply:
Foreign types are not always `*_type`, it could be configured by the `:foreign_type` option.
"both the foreign key and the foreign type" is preferable.

https://github.com/rails/rails/blob/260b273106168ba7a122c76c79a3b7f4a3209559/activerecord/test/models/post.rb#L143